docs/specification/draft/client/elicitation.mdx

```diff
@@ -151,139 +257,237 @@ To request information from a user, servers send an `elicitation/create` request
 }
 ```
 
-**Reject Response Example:**
+### URL Elicitation Requests
+
+URL elicitation enables servers to direct users to external URLs for out-of-band interactions that must not pass through the MCP client. This is essential for auth flows, payment processing, and other sensitive or secure operations.
+
+<Note>
+  **Important**: URL elicitation is *not* for authorizing the MCP client's
+  access to the MCP server (that's handled by [MCP
+  authorization](../basic/authorization)). Instead, it's used when the MCP
+  server needs to obtain sensitive information or third-party authorization on
+  behalf of the user. The MCP client's bearer token remains unchanged. The
+  client's only responsibility is to provide the user with context about the
+  elicitation URL the server wants them to open.
+</Note>
+
+URL elicitation requests **MUST** specify `mode: "url"` and include these parameters:
+
+| Name            | Type   | Description                               |
+| --------------- | ------ | ----------------------------------------- |
+| `url`           | string | The URL that the user should navigate to. |
+| `elicitationId` | string | A unique identifier for the elicitation.  |
+
+The `url` parameter **MUST** contain a valid URL. The `message` parameter **MUST NOT** contain a URL.
```

Comment:
We should mention _what kind_ of URL is valid here. Are these just HTTPS URIs? Can one use local URI schemes (e.g., `appx://`)? My initial sentiment is that we should restrict to HTTPS URIs, but maybe there are scenarios where local apps could be invoked (e.g., auth brokers)?

Reply:
HTTPS only is my initial sentiment as well (it can always be widened later, but narrowing later is hard).
@chhamilton and I discussed app-specific redirects in detail above.

The only open question in my mind is whether HTTP should be allowed (while HTTPS is still preferred) for local development/testing and stdio servers.